qiita_db/test/test_study.py

```diff
@@ -204,10 +204,8 @@ def _change_processed_data_status(self, new_status):
         # Change the status of the studies by changing the status of their
         # artifacts
         id_status = qdb.util.convert_to_id(new_status, 'visibility')
-        with qdb.sql_connection.TRN:
-            qdb.sql_connection.TRN.add(
-                "UPDATE qiita.artifact SET visibility_id = %s", (id_status,))
-            qdb.sql_connection.TRN.execute()
+        qdb.sql_connection.encapsulated_query(
+            "UPDATE qiita.artifact SET visibility_id = %s", (id_status,))
```

Comment:
Classic one-element tuple, maybe worthwhile converting it into a list? 

```suggestion
            "UPDATE qiita.artifact SET visibility_id = %s", [id_status])
```

Should be fine to disregard of course, I suggest using a list since that's what's mostly used in this PR.